components/styled/table/src/index.tsx

```diff
@@ -24,24 +24,14 @@ export const GenericTable = <TColumn, TRow>({
   rowData,
   orientation = 'VERTICAL',
   mapToTableData,
-}: IGenericTableProps<TColumn, TRow>) => {
-  if (!columnData || !rowData) {
-    return <div>Could not render table. Table columns or rows were empty.</div>
-  }
-
-  if (!mapToTableData) {
-    return <div>Could not render table. Mapping method not supplied.</div>
-  }
-
```

Comment:
These null checks aren't necessary since the parameters aren't nullable in the props